### PR TITLE
SLA: Focuse on the learner and Communicative competnece part

### PR DESCRIPTION
**Pull request by asiamac**
Second language acquisition/learning part 2

### CODE REVIEW

File: SLA.md [hunk 1]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
 The audiolingual method highlights the importance of speaking a language. With the use of **drills**, it systematically presents the succeeding elements of L2. Yule explains this method as a progression of certain language ‘habits’ through repeating oral exercises. However, some drawbacks of the audiolingual method are present. Firstly, as Yule claims, the deserted drills do not mirror the everyday usage of L2 and, moreover, this approach might become boring.   
 ### Communicative approaches
 Yule describes communicative approaches as more recent and focused primarily on the **functions** of language. This involves discussing the variety of **social settings** in classroom lessons.
-## Focus on the learner
 
+## Focus on the learner
+Recently, the field of L2 studies has changed the center of **interest** from teaching to **learner**. As a result, there is diffrenet approach to **errors**. They are no longer viewed negatively, but as **a sign of trying to progress**. Nowadays, communication is more valued than linguistic correctness.
```
Comment (brkutd) [inline SLA.md]: *diffrenet

File: SLA.md [hunk 2]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
...
 ### Transfer
-
+Transfer or *crosslinguistic influence* refers to phenomenon of influence of the L1 on the L2 in terms of analogical application of sounds, pharses and syntax. Copying from L1 similar lingustics features in L2 results in **postive transfer**, while transferring significantly different aspects leads to **negative transfer**.
```
Comment (brkutd) [inline SLA.md]: *pharses
*postive

File: SLA.md [hunk 3]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
...
 ### Interlanguage
-
+**Interlanguage** is a system with blended features from L1 and L2, which is fundamental in sceond language acqusition and production of L2. Process of **fossiilzation**, refering to the stage in which progress is no longer achieved, does not apply to the inrterlanguage.
```
Comment (brkutd) [inline SLA.md]: *sceond
*fossiilzation
*inrterlanguage

File: SLA.md [hunk 4]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
...
 ### Motivation
-
+Motivation is crucial factor in achieving success in second language acqusition. Among others, there is distinguished **instrumental motivation**, where learning L2 pursues a different goal, as well as **integrative motivation**, in which language learning is proceeded in order to achieve social goals.
 ### Input and output
-
+**Input**, which refers to learner's contact with the language, simplified in order to become more understandable is known as **foregin talk**. It allows learner to communicate, as well as presents clear examples of grammar, phrases and syntax  usage. Moreover, learner can gain L2 through **negotiated input**, which is gaining material during communication by asking questions for clarification while active attention is focused on what is stated.
```
Comment (brkutd) [inline SLA.md]: *syntax  usage (double space)

File: SLA.md [hunk 5]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
...
+### Task-Based learning
+**Task-based learning** helps to gain communicative commpetnece by providing variety of activities forcing learners to interact with each other.
 
 ## Communicative competence
-
-
-## Applied linguistics
+**Comunicative competence** is ability to communicate freely. It consists of three components:
```
Comment (brkutd) [inline SLA.md]: is -an- ability
*comunicative

File: SLA.md [hunk 6]
```diff
@@ -20,18 +20,23 @@ The grammar-translation method serves as **the most traditional** way of teachin
...
+* grammatical  competence,
+* sociolinguistic competence,
+* strategic competence.
+**Comunucation startegy** is a competence of coping with and bypassing comunication problems during conversation. The flexibility in language usage is essential for successful communication.
```
Comment (brkutd) [inline SLA.md]: *comunucation

File: SLA.md (hunk 1 above)
Comment (dsolska) [reply SLA.md]: *a different approach

File: SLA.md (hunk 2 above)
Comment (dsolska) [reply SLA.md]: *the phenomenon

File: SLA.md (hunk 3 above)
Comment (dsolska) [reply SLA.md]: *referring

File: SLA.md (hunk 6 above)
Comment (asiamac) [reply SLA.md]: corrected

File: SLA.md (hunk 5 above)
Comment (asiamac) [reply SLA.md]: corrected

File: SLA.md (hunk 3 above)
Comment (asiamac) [reply SLA.md]: corrected

File: SLA.md (hunk 2 above)
Comment (asiamac) [reply SLA.md]: corrected

File: SLA.md (hunk 1 above)
Comment (asiamac) [reply SLA.md]: corrected

File: SLA.md (hunk 4 above)
Comment (asiamac) [reply SLA.md]: please confirm that it is now ok

**Review by brkutd — APPROVED** [review]

**Review by dsolska — APPROVED** [review]

**Pull request merged**